### DISCUSSION TITLE
please add a shortcut to press ok in change window

### DISCUSSION THREAD

**Discussion by tildegrave** [monkeytypegame/monkeytype#5963] [Ideas] [answered]
### I would like to suggest adding a shortcut, such as `Ctrl+Enter` or `Shift+Enter` (*like in other apps*), to the OK button in the change window. This would streamline the workflow.
Thanks in advance.
![2024-10-10_01-02](https://github.com/user-attachments/assets/f769ae0b-68b3-4cdd-aa0a-b92c65c4a4fd)

**Comment by Miodec** [comment, marked as answer]
You can already press ctrl + enter to confirm this modal.

**Reply by tildegrave** [reply]
Ah, I see now. My custom keyboard layout, configured using xkb, maps the Enter key to the backslash. This is why the shortcut doesn't works. However, using the original Enter key works fine.
